### PR TITLE
Created README.md

### PR DESCRIPTION
**Pull request by jzipcode**
The basics for the README.md are here! I worked with the stuff we had, so most things are TBD yet. 
The file will be changed as the project is updated, and we can also add more screenshots/images (for instructions and stuff).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,46 @@
...
+## Getting Started
+
+### Prerequisites
+- A Discord account and server
+- TBD
+
+### Installation
+1. Clone the repository and navigate to the directory:
+```
+TBD
```
Comment (wgcv) [inline README.md]: We should add `git clone git@github.com:intelagense/howdy.git`

File: header-howdy.png
Comment (wgcv) [inline header-howdy.png:1]: I love the header!
We should put this in a folder, something like `/assets` or create a branch with and call the images from`/doc/img`?

**Comment by DaPandamonium** [comment]
@scorp-jzp maybe add also a structure example? For example one of my README.md:

## File Structure

```
sanctuary-bot/
│
├── bot.py
├── bluetracker.py
├── join_role.py
├── archive.py
├── requirements.txt
├── .env
├── last_pings.json
├── upcoming_boss_events.json
├── total_archived.json
├── pinned_threads.json
└── README.md
```

**Comment by intelagense** [comment]
Great idea @DaPandamonium 

Here is a useful tool I had bookmarked for that 😆 
https://tree.nathanfriend.io/

@scorp-jzp we don't have a file structure yet, but do you want to add a placeholder file structure with a title ?

File: README.md (hunk 1 above)
Comment (wgcv) [reply README.md]: Solved

**Review by wgcv — CHANGES_REQUESTED** [review]

File: header-howdy.png
Comment (intelagense) [reply header-howdy.png:1]: ➕ 1
We will probably refactor this when we get a file structure built. 👍

**Comment by intelagense** [comment]
@scorp-jzp 🙌 Thanks!

**Pull request merged**